hera_sim/tests/test_vis.py

```diff
@@ -45,6 +46,72 @@ def uvdataJD():
         start_time=2456659
     )
 
+def test_healvis_beam(uvdata, tmpdir):
+    freqs = np.unique(uvdata.freq_array)
+
+    # just anything
+    point_source_pos = np.array([[0, uvdata.telescope_location_lat_lon_alt[0]]])
+    point_source_flux = np.array([[1.0]] * len(freqs))
+
+    hv = HealVis(
+        uvdata=uvdata,
+        sky_freqs=np.unique(uvdata.freq_array),
+        point_source_flux=point_source_flux,
+        point_source_pos=point_source_pos,
+        nside=2**4
+    )
+
+    assert len(hv.beams) == 1
+    assert isinstance(hv.beams[0], healvis.beam_model.AnalyticBeam)
+
+    # Now try creating with an obsparam file
```

Comment:
These are different enough I would argue they should be different tests.

Reply:
yeah 'k/